### DISCUSSION TITLE
SVG support on Mac and Linux

### DISCUSSION THREAD

**Discussion by blockingHD** [ShapeCrawler/ShapeCrawler#823] [General]
Due to the use of SVG.Net for images, ShapeCrawler does not fully support Mac and Linux out-of-the-box. Looking at SVG.Net, it is possible to run on these x86 versions of these systems, but it requires extra setup https://svg-net.github.io/SVG/articles/Faq.html#im-getting-a-svggdipluscannotbeloadedexception-if-running-under-linux-or-macos.

Based on [this issue](https://github.com/CoreCompat/libgdiplus-packaging/issues/14) it seems ARM based macs (and likely any non x86 systems) are not supported. 

Either the support limitations should be documented, or a compatibility fix should be looked into.

**Comment by jcoliz** [comment]
Yikes. Not something easy to work around, either. PowerPoint wants a rasterized (bitmap) version of the SVG graphic, so full-fledged drawing is needed somehow.

Depending on how important this is, it should be possible to cleave out SVG capability into its own library, extending ShapeCrawler to allow plugins. Then at least linux/mac users could have everything but SVGs.
